sdk/aqueduct/aqueduct_client.py

```diff
@@ -88,10 +90,12 @@ def __init__(
         """
         logging.basicConfig(level=log_level)
         self._api_client = APIClient(api_key, aqueduct_address)
+        api_client.__GLOBAL_API_CLIENT__ = self._api_client
         self._connected_integrations: Dict[
             str, IntegrationInfo
         ] = self._api_client.list_integrations()
         self._dag = DAG(metadata=Metadata())
+        dag.__GLOBAL_DAG__ = self._dag
```

Comment:
the global api client is fine with me. But what happens if we user constructs multiple aqueduct clients? Do we want to wipe out the previous dag completely? I'd rather us not overwrite the global dag if it already exists, so we'd have two clients using the same underlying dag, which is weird but seems less destructive.